### ISSUE TITLE
Is this still current?

### ISSUE THREAD

**Issue by kdesigns** [aphichat/cartthrob-mollie-ideal#2]
Does this still work?

**Comment by aphichat** [comment]
Yes its still working, but with the right CT version and EE version(not 3+) feel free to creat a PR or modify the code as you want.
